php

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
I like this script, but have some comments below.

Reply:
btw, once we can make use of the -w option for docker-compose exec, there will no need for having two scripts (script and php). Just php should be sufficient. Maybe script should be renamed to php-in-mediawiki-home or something.

Reply:
What's the use case for invoking `php` on something that isn't somewhere under `/var/www/mediawiki`?